vertical-pod-autoscaler/hack/kind-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+featureGates:
+  InPlacePodVerticalScaling: true
```

Comment:
should we update with the new name?

Reply:
That feature gate is for the KEP-1287 API to work.